cmd/heznode/cfg.builder.toml

```diff
@@ -17,6 +17,7 @@
 #MaxSQLConnections = 100
 ### Maximum amount of time that an API request can wait to establish a SQL connection
 #SQLConnectionTimeout = "2s"
+#CoordinatorNetwork = false
```

Comment:
Here the example would be true. Also, I am missing the FindPeersCoordinatorNetworkInterval = "300s" parameter